contracts/partial/yToken/lendingTypes.ligo

```diff
@@ -134,8 +151,8 @@ type calculateCollParams is [@layout:comb] record [
   res                   : nat;
 ]
 
-type transferType is TransferOutside of faTransferParams
-type iterTransferType is FA2TransferOutside of transferParams
+type transferType is TransferOutside of fa12TransferParams
+type iterTransferType is FA2TransferOutside of fa2TransferParams
```

Comment:
Any chance we can come with better names for these?
For example
`fa12TransferType`
and
`fa2TransferType`